region/lowernorfair/east.json

```diff
@@ -5967,7 +5967,7 @@
                   "requires": [
                     "h_canNavigateHeatRooms",
                     "SpeedBooster",
-                    "canCrumbleSpinJump",
+                    "canCrumbleJump",
                     "canTrickyDashJump",
```

Comment:
How hard is this strat? It sounds like it could be notable, although, it doesn't occur very often.

Reply:
Its about exactly as hard as `canCrumbleJump` + `canTrickyDashJump`.  The crumbles help time the jump correctly, because you know you have no time already to jump on crumbles.  But the crumbles also limit the number of attempts you have at the trick by falling eventually.  Notable makes sense.